pkg/messaging/rabbitmq/publisher.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package rabbitmq
+
+import (
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/mainflux/mainflux/pkg/messaging"
+	amqp "github.com/rabbitmq/amqp091-go"
+)
+
+var _ messaging.Publisher = (*publisher)(nil)
+
+type publisher struct {
+	connection *amqp.Connection
+	channel    *amqp.Channel
```

Comment:
Since this is rather a simple struct, consider using `conn` and `ch` instead.

Reply:
Done